arrocke/gloss-translation
# feat: language users

## What has changed



* Add language users
  * Database has three roles - viewer, translator, and admin
  * API and UI consider a user a member of a language if they have the viewer role
  * Members can additionally have either the translator or admin role, or both.
* Added multiselect input component

There is still more work to do in other PRs:
* Edit language roles
* Remove users
* Give UI access to language admins

## Connected Issues



closes #68

## QA Steps



1. Log in as a platform admin
2. Navigate to languages list
3. Open the manage page of a language
4. Add a user and give them a role
5. Check that user is created and added to language

## Post-Deployment
